code/modules/mob/living/carbon/human/species_types/mothmen.dm

```diff
@@ -78,23 +78,25 @@
 
 /datum/species/moth/get_species_lore()
 	return list(
-		"Their homeworld lost to the ages, the moths live aboard the Grand Nomad Fleet. \
-		Made up of what could be found, bartered, repaired, or stolen the armada is a colossal patchwork \
-		built on a history of politely flagging travelers down and taking their things. Occasionally a moth \
-		will decide to leave the fleet, usually to strike out for fortunes to send back home.",
-
-		"Nomadic life produces a tight-knit culture, with moths valuing their friends, family, and vessels highly. \
-		Moths are gregarious by nature and do best in communal spaces. This has served them well on the galactic stage, \
-		maintaining a friendly and personable reputation even in the face of hostile encounters. \
-		It seems that the galaxy has come to accept these former pirates.",
-
-		"Surprisingly, living together in a giant fleet hasn't flattened variance in dialect and culture. \
-		These differences are welcomed and encouraged within the fleet for the variety that they bring.",
+		"The native language of Gamuoida is referred to as 'Gamuid' by their population. However, 'Moffic' is used as an exonym among other species.",
+
+		"Written transcripts of Gamuoida history remain largely untranslated to modern Gamuid, that most of the populace knows. \
+		However, more recent transcripts detail the progress of Gamuoida working to accelerate the process of first contact - to get in touch with extra-terrestrials. \
+		There is a long history of research and scientific development done by Gamuoida, with lower-class Gamuoida focusing on providing materials and other needed produce for upper-class researchers to utilize in advancements. ",
+
+		"Upon meeting other lifeforms outside of their planet, notably Humans, the Gamuoida would begin to offer their population as a labor force for stations, ships, and other facilities requiring employment. \
+		There wasn't a fixed paycheck at first for moths when they first began work. This led to an economic crisis and a huge spike in inflation on their homeworld when converting credits into their currency, the Posis, \
+		With some careful consideration, Gamuoida working on Nanotrasen facilities are paid 20% less than most, and are given a weekly lamp to supplement the unpaid money. \
+		This has pleased 98% of surveyed moths, including the upper class.",
+
+		"The working, lower, and upper classes of Gamuoida are scattered across their homeworld of Paraco, including other under-studied planets, with the wealthy more focused on settling down in rain forests, primarily to take shelter in tall trees. \
+		The variety in weather and temperature endemic to Paraco has given rise to a vast amount of clothing styles, with a key focus on embroidered designs on cloth, \
+		though most often those working on stations and other facilities wear heavy-duty equipment or their assigned uniforms.",
```

Comment:
"Gamuoida" should be "Gamuioda" (`uoi`>`uio`) in some places, and there's an extra space at the end of line 90. LGTM otherwise. 👍
```suggestion
		"The native language of Gamuioda is referred to as 'Gamuid' by their population. However, 'Moffic' is used as an exonym among other species.",

		"Written transcripts of Gamuioda history remain largely untranslated to modern Gamuid, that most of the populace knows. \
		However, more recent transcripts detail the progress of Gamuioda working to accelerate the process of first contact - to get in touch with extra-terrestrials. \
		There is a long history of research and scientific development done by Gamuioda, with lower-class Gamuioda focusing on providing materials and other needed produce for upper-class researchers to utilize in advancements.",

		"Upon meeting other lifeforms outside of their planet, notably Humans, the Gamuioda would begin to offer their population as a labor force for stations, ships, and other facilities requiring employment. \
		There wasn't a fixed paycheck at first for moths when they first began work. This led to an economic crisis and a huge spike in inflation on their homeworld when converting credits into their currency, the Posis, \
		With some careful consideration, Gamuioda working on Nanotrasen facilities are paid 20% less than most, and are given a weekly lamp to supplement the unpaid money. \
		This has pleased 98% of surveyed moths, including the upper class.",

		"The working, lower, and upper classes of Gamuioda are scattered across their homeworld of Paraco, including other under-studied planets, with the wealthy more focused on settling down in rain forests, primarily to take shelter in tall trees. \
		The variety in weather and temperature endemic to Paraco has given rise to a vast amount of clothing styles, with a key focus on embroidered designs on cloth, \
		though most often those working on stations and other facilities wear heavy-duty equipment or their assigned uniforms.",
```